template/module/readme/CONTEXT.md

```diff
@@ -1,13 +1,16 @@
 [ This file is optional but strongly suggested to allow end-users to evaluate the
 module's usefulness in their context. ]
 
-It should explain the “why” of the module:
-
-- in which context or use cases this module can be useful (practical examples are welcome!).
+BUSINESS NEED: 
```

Comment:
I worry a little bit that people will misinterpret these capitalized words as subtitles they have to copy in their actual documentation, which would be a little bit strange.

Reply:
Those sections are examples of how to structure the Context Fragment, not mandatory content.
I find id really helpful because sometimes you just don't know where to begin.